test/integration/render/scenarios/regression/mvt-interactivity/scenario.js

```diff
@@ -0,0 +1,76 @@
+const map = new mapboxgl.Map({
+    container: 'map',
+    style: { version: 8, sources: {}, layers: [] },
+    center: [0, 0],
+    zoom: 0
+});
+
+const METADATA = {
+    properties: {
+        cartodb_id: { type: 'number' },
+        winner: { type: 'category' }
+    },
+    idProperty: 'winner' // 'cartodb_id'
+};
+
+const MVT_FILE = '/test/integration/user/test/data/5_15_10.mvt';
+const source = new carto.source.MVT(MVT_FILE, METADATA);
+const viz = new carto.Viz(`
+    @f_cartodb_id: $cartodb_id
+    @f_winner: $winner
+
+    @v_features: viewportFeatures($winner)
+    strokeWidth: 1
+`);
+const layer = new carto.Layer('my_mvt_layer', source, viz);
+
+const interactivity = new carto.Interactivity(layer);
+
+interactivity.on('featureClick', async (event) => {
+    const aFeature = event.features[0];
+
+    console.log(aFeature.id);
```

Comment:
I'm gonna remove these lines.

Reply:
The original problem was related to `id` properties, so that's why I left that complementary logs.